File: src/main/java/org/activecraft/activecraftcore/listener/ServerPingListener.java
package org.activecraft.activecraftcore.listener;

import org.activecraft.activecraftcore.ActiveCraftCore;
import org.activecraft.activecraftcore.utils.config.MainConfig;
import org.activecraft.activecraftcore.ActiveCraftCore;
import org.activecraft.activecraftcore.utils.config.MainConfig;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.server.ServerListPingEvent;

public class ServerPingListener implements Listener {

    MainConfig mainConfig = ActiveCraftCore.getInstance().getMainConfig();

    @EventHandler
    public void on(ServerListPingEvent event) {
        //set modt when locked down
        if (mainConfig.isLockedDown()) {
            mainConfig.set("old-modt", event.getMotd());
            event.setMotd(mainConfig.getLockdownModt());
        } else {
            event.setMotd(mainConfig.getOldModt());
        }
    }
}

File: src/main/java/org/activecraft/activecraftcore/listener/ServerPingListener.kt
package org.activecraft.activecraftcore.listener

import org.activecraft.activecraftcore.ActiveCraftCore
import org.activecraft.activecraftcore.utils.config.MainConfig
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.server.ServerListPingEvent

class ServerPingListener : Listener {
    var mainConfig: MainConfig = ActiveCraftCore.INSTANCE.mainConfig
    @EventHandler
    fun on(event: ServerListPingEvent) {
        if (mainConfig.lockedDown) {
            mainConfig["old-modt"] = event.motd
            event.motd = mainConfig.lockdownModt!!
        } else {
            event.motd = mainConfig.oldModt!!
        }
    }
}